_includes/header.html

```diff
@@ -13,9 +13,9 @@ <h1 class="big">
             <li>
                 <a href="/code">Code</a>
             </li>
-            <!-- <li> -->
-            <!-- <a href="/blog">Blog</a> -->
-            <!-- </li> -->
+            <li>
```

Comment:
Can we keep this commented out when we merge this PR? Now that this has been deployed on hack.gt, there are _real_ people visiting the site, so we don't want fake/test content easy for outside people to access.